src/cbs_assignment.hpp

```diff
@@ -0,0 +1,224 @@
+#pragma once
+
+#include <boost/bimap.hpp>
+#include <boost/graph/adjacency_list.hpp>
+// #include <boost/graph/find_flow_cost.hpp>
+#include <boost/graph/successive_shortest_path_nonnegative_weights.hpp>
+
+namespace libMultiRobotPlanning {
+
+/*!
+  \example assignment.cpp example that takes cost mappings from a file
+*/
+
+/*! \brief Find optimal (lowest total cost) assignment
+
+This class can find the lowest sum-of-cost assignment
+for given agents and tasks. The costs must be integers, the agents and
+tasks can be of any user-specified type.
```

Comment:
Update this part. In particular, how is CBS_assignment different from assignment?